Magento_Catalog/web/template/product/list/columns/image_with_borders.html

```diff
@@ -5,12 +5,19 @@
         attr="href: $row().url"
     >
         <div class="product-grid-item__image-wrapper">
-            <div class="lazyload-wrapper">
+            <div
+                class="
+                    lazyload-wrapper
+                    ratio-container
+                "
+                data-bind="style: { paddingBottom: getPaddingBottom(getWidth($row()), getHeight($row())) }"
```

Comment:
do we need a mixin for this? what do you think about doing the calculation here?
``${(getHeight($row()) / getWidth($row()) * 100}%`
it would be one js file less